rest/resources/rest.json

```diff
@@ -40,7 +40,9 @@
     "timeout": 1000,
     "tlsClientCertificatePath": "/",
     "tlsClientKeyPath": "/",
-    "tlsCaCertificatePath": "/"
+    "tlsCaCertificatePath": "/",
+    "networkPropertyFilePath": "/",
+    "nodePropertyFilePath": "/"
```

Comment:
Remove old propertiesFilePath from network?

Reply:
the old one is not even in the config, :)